improver_tests/acceptance/test_estimate_samos_gams.py

```diff
@@ -131,6 +135,8 @@ def test_gam_at_sites():
         gam_features,
         "--model-specification",
         model_specification_path,
+        "--window-length",
```

Comment:
Although this test is passing, I'm currently getting a checksum failure in the SHA256SUMS file.

Reply:
The tests all pass for me with the latest version of master (after this PR has been merged: https://github.com/metoppv/improver_test_data/pull/108).